src/coder_runtime.cpp

```diff
@@ -3801,7 +3801,7 @@ namespace coder
                     const octave_value& tmp = rhs_val (k);
 
                     if (tmp.is_undefined ())
-                      error ("element number %d undefined in return list",
+                      error ("element number %lld undefined in return list",
```

Comment:
mingw won't understand this format specifier.
Please consider using the PRId64 macro: http://www.cplusplus.com/reference/cinttypes/

Reply:
Thanks for pointing this out.
Would it be OK to use :

error ("element number %s undefined in return list", std::to_string (k+1).c_str ());

instead?

Reply:
That sounds reasonable to me as well.